sdk/cognitivelanguage/Azure.AI.Language.Text.Authoring/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Release History
```

Comment:
Update change log

Reply:
Gouri, thanks for commenting! Since this Text Authoring SDK is the first version, it might not need ChangeLog here?